src/bbmustache.erl

```diff
@@ -289,6 +289,10 @@ compile_impl([{'#', Keys, Tags, Source} | T], Data, Result, State) ->
                                              Result, Value), State);
       _ when Value =:= false ->
             compile_impl(T, Data, Result, State);
+      _ when Value =:= nil ->
```

Comment:
Does Elixer commonly use nil?
If so, please provide links that it can refer to.

NULL has various notations depending on the language, so I think that it will necessary to consider the validity.

Reply:
Yes, Elixir's nil is equivalent to NULL in most languages I'm familiar with. https://hexdocs.pm/elixir/Kernel.html#module-truthy-and-falsy-values